extensions/wikia/AssetsManager/config.php

```diff
@@ -817,12 +817,14 @@
 		'//extensions/wikia/AdEngine/js/provider/gpt/googleTag.js',
 		'//extensions/wikia/AdEngine/js/provider/gpt/helper.js',
 		'//extensions/wikia/AdEngine/js/provider/paidAssetDrop.js',
-		'//extensions/wikia/AdEngine/js/recovery/helper.js',
 		'//extensions/wikia/AdEngine/js/provider/remnantGptMobile.js',
 		'//extensions/wikia/AdEngine/js/slot/adSlot.js',
 		'//extensions/wikia/AdEngine/js/utils/AdLogicZoneParams.js',
 		'//extensions/wikia/AdEngine/js/utils/hooks.js',
 
+		// Recovery
+		'//extensions/wikia/ARecoveryEngine/js/helper.js',
```

Comment:
Just wonder what will happen on mercury when blocking is detected (there is no SourcePointTag).